x-pack/dockerlogbeat/magefile.go

```diff
@@ -374,6 +375,9 @@ func GolangCrossBuild() error {
 
 // Package builds a "release" tarball that can be used later with `docker plugin create`
 func Package() {
+	if devtools.FIPSBuild && !slices.Contains(devtools.FIPSConfig.Beats, "dockerlogbeat") {
```

Comment:
what is `dockerlogbeat`?

Reply:
A beat that is not ready for FIPS but overrides the `Package()` mage target.
This modification is just a best effort to allow invoking `FIPS=true make release` from the root of the repo and have all the FIPS compliant beats packages generated.
Dockerlogbeat redefining the packaging forced me to add this exclusion to avoid build errors (btw it was the only one that still broke even after all the filtering done for build/packaging beats).